app/components/DeleteTodo.tsx

```diff
@@ -0,0 +1,21 @@
+"use client"
+
+import { useRouter } from "next/navigation"
+
+export default function DeleteTodo(props:any) {

```

Comment:
Don't use any as type and also restructure the props object and read author and id 
``` typescript
export default function DeleteTodo({author , id} : {author: string, id : string})
```